go.mod

```diff
@@ -40,6 +38,7 @@ require (
 
 /// Pinned to kubernetes-1.16.2
 replace (
+	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
```

Comment:
Where is this coming from? operator-sdk?

Reply:
changed to docker to make it clearer